src/main/java/de/qabel/desktop/ui/transfer/FxProgressModel.java

```diff
@@ -28,7 +39,15 @@ public void setProgress(HasProgress progress) {
 		updateProgress();
 	}
 
+	long lastUpdate = 0;
+
 	private void updateProgress() {
-		Platform.runLater(() -> progressProperty.set(this.progress.getProgress()));
+		final double progress = this.progress.getProgress();
+		long now = System.currentTimeMillis();
+		if (progress != 1.0 && minimumUpdateDelay != null && now < lastUpdate + minimumUpdateDelay) {
```

Comment:
maybe too much abstraction ... even >= 1 wouldn't help. T should be required to implement isDone or similar